engines/registration_sharing/lib/registration_sharing/client.rb

```diff
@@ -21,7 +21,7 @@ def sync_system
   def peer_register_system(system)
     params = {}
 
-    %w[login password hostname proxy_byos system_token registered_at created_at last_seen_at instance_data].each do |attribute|
+    %w[login password hostname proxy_byos proxy_byos_mode system_token registered_at created_at last_seen_at instance_data].each do |attribute|
```

Comment:
I would have expected some code in the registration sharing that translates from `proxy_byos=true` to `proxy_byos_mode: :byos`? So that when an older RMT sends that value, still the new field gets updated.

Reply:
:white_check_mark: